misc/setwall.py

```diff
@@ -60,22 +60,26 @@
     "alternating_tint_fg" : "#000000ff",
 }
 
+
 def set_wallpaper(img_path):
     subprocess.run(["feh", "--bg-fill", "--no-fehbg", img_path])
 
+
 def get_avg_luminance(img_path):
-    im = Image.open(image_path)
```

Comment:
Not sure here as the variable image_path is at the end of the script. But as the `get_avg_luminance` is called with argument and `img_path` was not used in the function, I assumed that `image_path` should be changed to `img_path`.

Reply:
Good call! It's funny that the script still worked this way.